hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/config/GsonConfig.kt

```diff
@@ -17,40 +17,33 @@ import java.time.format.DateTimeFormatter
 import java.util.Locale
 
 @Configuration
-class GsonConfig() {
+class GsonConfig {
 
   @Bean
-  fun gson(): Gson {
-    return GsonBuilder().setPrettyPrinting()
-      .registerTypeAdapter(LocalDate::class.java, LocalDateAdapter())
-      .registerTypeAdapter(LocalDateTime::class.java, LocalDateTimeAdapter())
-      .create()
-  }
+  fun gson(): Gson = GsonBuilder().setPrettyPrinting()
+    .registerTypeAdapter(LocalDate::class.java, LocalDateAdapter())
+    .registerTypeAdapter(LocalDateTime::class.java, LocalDateTimeAdapter())
+    .create()
```

Comment:
I thnk Pete moved the indexer away from Gson to Jackson so maybe we should do the same at some point

Reply:
yes - I agree. Cheers